ubuntu/standard/Dockerfile.template

```diff
@@ -140,6 +140,11 @@ RUN apt-get update \
     python3-pip \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Ruby
+RUN apt-get update \
+  && apt-get install -y ruby ruby-dev ruby-bundler \
```

Comment:
nit: remove one space at the beginning of this line and the next so the commands are aligned.

Reply:
👍